internal/provider/metric_export_prometheus_config_resource_test.go

```diff
@@ -0,0 +1,213 @@
+package provider
+
+import (
+	"context"
+	"fmt"
+	"github.com/cockroachdb/cockroach-cloud-sdk-go/pkg/client"
+	mock_client "github.com/cockroachdb/terraform-provider-cockroach/mock"
+	"github.com/golang/mock/gomock"
+	"github.com/google/uuid"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"log"
+	"net/http"
+	"os"
+	"testing"
+)
+
+// TestAccMetricExportPrometheusConfigResource attempts to create, check, and destroy
+// a real cluster. It will be skipped if TF_ACC isn't set.
+func TestAccMetricExportPrometheusConfigResource(t *testing.T) {
+	t.Skip("Skipping until we can either integrate the AWS provider " +
+		"or import a permanent test fixture.")
+	t.Parallel()
+	clusterName := fmt.Sprintf("%s-prometheus-%s", tfTestPrefix, GenerateRandomString(4))
```

Comment:
I've been trying to keep these names short so an org that runs the test doesn't need to enable the feature flag to allow long names.  

the test prefix is 6
-prometheus- is 12
and random is 4

The name limit is 20. 

How about using `"%s-prom-exp-%s"` instead?

Reply:
This block has been removed. Refer to the reply on the previous comment.